src/cluster.cpp

```diff
@@ -432,9 +432,16 @@ void Cluster::fillTestData()
 }
 #endif
 
-void Cluster::fillData(const std::string& answerfilePath)
+void Cluster::dumpData(const std::string& answerfilePath)
 {
+    AnswerFile answerfile(answerfilePath);
+    LOG_TRACE("Dump Management Network");
+
+    answerfile.dumpFile(answerfilePath)
+}
```

Comment:
I didn't understand where the data dumped into the file was coming from. I guess it has to come from the `cluster` instance, right?

Reply:
Yes, but I am coding the part that saves it right now.